src/linter/fileTypes/linter.ts

```diff
@@ -0,0 +1,39 @@
+import {Resource} from "@ui5/fs";
+import {LinterParameters} from "../LinterContext.js";
+import {MESSAGE} from "../messages.js";
+
+const deprecatedViewFileTypes = [
+	".view.json",
+	".view.html",
+	".view.js",
+	".view.tmpl",
+	".view.ts",
+	".fragment.html",
+	".fragment.js",
+	".fragment.ts",
+];
+
+export default async function lintFileTypes({workspace, context}: LinterParameters) {
+	let xmlResources: Resource[];
```

Comment:
To me this var name seems a bit ambiguous as it refers to XMLResources, but what we're looking for is anything, but XML :D 

Maybe, `potentialDeprecatedResources` or something in that direction

Reply:
Duplicate comment.